test/org/sagebionetworks/bridge/play/controllers/AuthenticationControllerMockTest.java

```diff
@@ -79,12 +83,18 @@
     @Mock
     CacheProvider cacheProvider;
     
+    @Mock
+    UserSession userSession;
```

Comment:
Similarly, UserSession is also just getters and setters. We should use a concrete one instead of mocking it.